soccer/src/soccer/strategy/coach/coach_node.hpp

```diff
@@ -35,12 +36,12 @@ enum MatchSituation {
     in_play,         // normal play
 };
 
-enum Positions { Goalie, Defense, Offense, PenaltyPlayer, GoalKicker };
+enum Positions { Goalie, Defense, Offense, Runner, PenaltyPlayer, GoalKicker };
 
 // These values are explicitly declared because they are the ints that are published to
 // strategy/positions i.e. the same values as strategy::Positions
 namespace OverridePosition {
-enum OverridePosition { Goalie = 0, Defense = 1, Offense = 2, None = 3 };
+enum OverridePosition { Goalie = 0, Defense = 3, Offense = 2, Runner = 1, None = 4 };
```

Comment:
changing this doesn't do anything; it's for the UI tab to assign positions (which is broken anyway, LOL)